src/components/pages/datasets/pep/PepList.jsx

```diff
@@ -20,10 +20,10 @@ const PepList = () => {
     },
     {
       header: t('fullName'),
-      prop: 'fullname',
+      prop: i18n.language === 'uk' ? 'fullname' : 'fullname_en',
       width: '20%',
       filter: {
-        name: 'fullname',
+        name: i18n.language === 'uk' ? 'fullname' : 'fullname_en',
```

Comment:
we dont have filter for `fullname_en`

Reply:
filter added in https://github.com/3v-workspace/Data_converter/pull/880